specs/regional_communities.md

```diff
@@ -1,61 +1,333 @@
+- Select you language, which updates the available communities. Default language is English.
+- Any combination of the above will further narrow the search
+
+
+### Approve a new Community member
+Approvind a new community member could be as simple as a "follow back" by the owner of the community or someone who has been given the keys to control the community account. 
+
+It is recommended that only human-verified accounts be allowed to become members, as a minimum requirement.
+
+A drawback of this approach is that there is no vote. Any single admin can follow or unfollow, which may be too arbitrary for some communities. Another drawback is that the "follow" feature would no longer be available to use for its originally intended purpose. A more structured approach to membership may have to be contemplated. Optins include:
+- user follows the Community and automatically becomes a member as long as some criteria are filled (e.g. is human verified and accepts the code of conduct) => Preferred
+- user applies for membership, followed by a majority vote to approve the application
+- membership is open to anyone without approval
+- membership is open to anyone, with the restriction that they can be a full member of one and only one community
+
+This author recommends a combination, whereby there are four types of membership:
+1. Home Membership - each user can define their one and only "home" membership = the community that is most closely aligned with them. This can be done without approval from the community, but each user can only select exactly one home community (or not make the selection).
+2. Hangaroud Membership - any user can be a hangaround. This feature is similar to a standard "follow" and should be the default membership for most communities.
+3. Core Member - the user has been approved in some way (by community vote or admin vote) to be a core member of the community. This option should not be a default option, but could be a possibility to enable for some communities that desire this option.
+4. Admin Member - the user has admin rights for the group
+
+It is this authors recommendation that commuinties be open for anyone to join and that applications are "accepted by default". If limitations are desired then it's better to use the "silence the member" feature than to go to "accepted pending approval" which signals a closed or restricted society.
+
+### Split a Community
+Instead of creating a Community from scratch, splitting an existing Community into two should be the default way of creating new communities. This is a core feature of the Community widget design.
+
+When splitting a community, the key question to ask the user is this: "Along which dimension do you want the split to occur: Geography, Language, or Constellation?"
+- If splitting along the Geography dimension then ask also if it's a sibling-split of child-split. 
```

Comment:
who and how verifies the geography dimension? full KYC may be tricky and will require users to pay.

Reply:
Either first one to suggest a new geo-split gets to do it. Or we assign a panel to curate and approve it. I like the idea of free assignment, combined with letting communities choosing the parent they most align with, for a naturally evolving meta-hierarchy.